app/components/FailedIdentityModal.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { removeIdentityAndInitialAccount } from '~/features/IdentitySlice';
+import ChoiceModal from '~/components/ChoiceModal';
+import routes from '~/constants/routes.json';
+import { initialAccountNameSelector } from '~/features/AccountSlice';
+
+interface Props {
+    identityId: number;
+    identityName: string;
+    isRejected: boolean;
+}
+
+/**
+ * Modal, which appears on rejected identities/initial accounts.
+ */
+export default function FailedIdentityModal({
+    identityId,
+    identityName,
+    isRejected,
+}: Props) {
+    const dispatch = useDispatch();
+    const [modalOpen, setModalOpen] = useState(false);
+    const initialAccountName = useSelector(
+        initialAccountNameSelector(identityId)
+    );
+
+    useEffect(() => {
+        if (isRejected) {
+            setModalOpen(true);
+        } else {
+            setModalOpen(false);
+        }
+    }, [isRejected, setModalOpen]);
+
+    async function remove() {
+        await removeIdentityAndInitialAccount(dispatch, identityId);
+        setModalOpen(false);
+    }
+
+    return (
+        <ChoiceModal
```

Comment:
On the `Modal` component, it's possible to set a `disableClose` prop, which I think would fit this use case. When the close button is there, it's possible for users to opt out of removing the identity/initial account.

Reply:
I've used this now, as pressing 'later' does the same.